docker-compose.yml

```
@@ -0,0 +1,13 @@
+services:
+  #pgdb-dev
+  postgresql-dev:
+    image: postgres
+    container_name: "pgdb-dev"
+    restart: always
+    shm_size: 128mb
+    environment:
+      POSTGRES_PASSWORD: "${DATABASE_PASSWORD}"
+      POSTGRES_DB: "${POSTGRES_DB}"
+    ports:
+      - "5432:5432"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)